_posts/01-02-01-Use-the-Current-Stable-Version.md

```diff
@@ -1,15 +1,16 @@
 ---
-title:   Use the Current Stable Version (5.6)
+title:   Use the Current Stable Version (7.0)
 isChild: true
 anchor:  use_the_current_stable_version
 ---
 
-## Use the Current Stable Version (5.6) {#use_the_current_stable_version_title}
+## Use the Current Stable Version (7.0) {#use_the_current_stable_version_title}
 
-If you are getting started with PHP, start with the current stable release of [PHP 5.6][php-release]. PHP has added 
-powerful [new features](#language_highlights) over the last few years. Though the incremental version number difference 
-between 5.2 and 5.6 is small, it represents _major_ improvements. If you are looking for a function or its usage, the 
-documentation on the [php.net][php-docs] website will have the answer.
+If you are getting started with PHP, start with the current stable release of [PHP 7.0][php-release]. PHP 7.0 is very
+new, and adds many amazing [new features](#language_highlights) over the older 5.x versions. The engine has been largely re-written, and PHP is now even quicker than older versions.
```

Comment:
This will need to have some language highlights for the new functionality. I have started a thing for type hints, explaining the history and the new usage, but happy to get more on other bits.